libsolidity/analysis/DocStringTagParser.cpp

```diff
@@ -165,12 +165,7 @@ void DocStringTagParser::parseDocStrings(
 			returnTagsVisited++;
 			if (auto const* varDecl = dynamic_cast<VariableDeclaration const*>(&_node))
 			{
-				if (!varDecl->isPublic())
-					m_errorReporter.docstringParsingError(
-						9440_error,
-						_node.documentation()->location(),
-						"Documentation tag \"@" + docTag.first + "\" is only allowed on public state-variables."
-					);
+				solAssert(varDecl->isPublic(), "@param is only allowed on public state-variables.");
```

Comment:
It's actually neither valid on public nor private state variables.

Reply:
Sorry, typo. It was meant to be `@return`, not `@param`.